src/Raven.Server/Documents/Handlers/Debugging/DatabaseDebugInfoPackageHandler.cs

```diff
@@ -16,7 +16,7 @@ public class DatabaseDebugInfoPackageHandler : DatabaseRequestHandler
         [RavenAction("/databases/*/debug/info-package", "GET", AuthorizationStatus.ValidUser, IsDebugInformationEndpoint = true)]
         public async Task GetInfoPackage()
         {
-            var contentDisposition = $"attachment; filename=debug-info of {Database.Name} {DateTime.UtcNow}.zip";
+            var contentDisposition = $"attachment; filename={DateTime.UtcNow:yyyy-MM-dd H:mm:ss}.zip";
```

Comment:
Should be at the end of the file name, but we do need it to tell where this came from.